pep/app/services/current-user.ts

```diff
@@ -54,12 +54,10 @@ export default class CurrentUserService extends Service {
      * @returns Promise<User | void>
      */
     async fetchUser(): Promise<User | void> {
-        if (this.session.isAuthenticated) {
-            const { SessionId } = this.session.data.authenticated;
-            const user = await this.store.queryRecord('user', { SessionId });
-            this.user = user;
-            return user;
-        }
+        const { SessionId } = this.session.data.authenticated;
+        const user = await this.store.queryRecord('user', { SessionId });
```

Comment:
not sure if @adistasio every fully confirmed this with them. but I think we need to add the `userType` field to the user model, and then update the `updatePrefs()` method so that the api User record is NOT updated unless the `userType ===  "Individual"` (or `!== "Group"`, might be better depending on what the eventual user type will be for federated/saml logins). 

Because even if the preferences update endpoint is working for you when on an IP login, this would be bad, because all users on that IP would be sharing those same preferences -- so pref changes should be local/cookie-stored only.

Reply:
Is Individual or Group coming back?

Reply:
I saw it in the swagger docs I think, but I'm not 100% sure. @bakerac4 when you get a successful IP auth response, does it actually have `UserType: "Group"` in the data?

Reply:
Rather, in the GET /Users?SessionId= call is where it should be.

Reply:
I don't have that branch running right now, but Im 90% sure it did - yes